src/app/screens/rareSatsBundle/index.tsx

```diff
@@ -152,7 +176,11 @@ function RareSatsBundle() {
   useResetUserFlow('/rare-sats-bundle');
 
   const handleBackButtonClick = () => {
-    navigate('/nft-dashboard?tab=rareSats');
+    if (source === 'OrdinalDetail') {
+      navigate(-1);
+    } else {
```

Comment:
because if the user opens this screen from the 'open in gallery' button, the new tab will open with no history at -1, and this will throw an exception.

in other screens, I implemented a hideBackButton if no history to go back to

Reply:
is there an example of that in the code?